docs/blog/newsletter-february-2020.rst

```diff
@@ -0,0 +1,96 @@
+.. post:: February 03, 2020
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – February 2020
+#########################################
+
+Happy 2020, documentarians! It's Beth and the newsletter crew here, with the latest and greatest from the Write the Docs community.
+
+Two bits and pieces for you on the news front today. In December we annouced dates for the Prague conference, which will be 13-15 September - and in a brand shiny new venue, the Hotel Olšanka! `Read all about it here </conf/prague/2020/news/init/>`__. As for across the pond, the speakers for Portland speakers will be announced soon, so keep your eyes peeled for those. The conference is 3-5 May, and `tickets are on sale right now </conf/portland/2020/tickets/>`__.
+
+So with that, let's see what's been cooking on Slack recently!
+
+-----------------------------
+#learn-tech-writing book club
+-----------------------------
+
+There's been a buzz of activity all through January in the `#learn-tech-writing <https://app.slack.com/client/T0299N2DL/C7YJR1N02>`__ channel, where folks have been discussing *The Product is Docs* chapter-by-chapter. As those involved have kindly started an effort to write up all the fantastic conversations on the topic, we're not going to write it up in the newsletter, but we wanted to point you towards what's been written up so far:
```

Comment:
lol the hyphenated construction which was ALSO discussed on Slack recently (or maybe it was on work Slack, I can't keep track anymore ...) -- anyway, pretty sure it's "chapter by chapter" w/o hyphens, although that's a nit that you can ignore if you like.
That's also three instances of "write up" and variants in a single sentence. Maybe something more like"
"Participants have started a writeup of all the fantastic conversations the discussion produced, so we won't duplicate their efforts here. But here's a list of what they've produced so far:"

Reply:
Your version is far better, added